File: DAGF-AlgortimoProgramacaoDeComputadores-exercicios/cap6/multiplos.c
#include <stdio.h>

int main(){
    int a,b,troca,multiplo;
    
    printf("\nWADAJU SOFTWARE SOLUTIONS\n");
    printf("Multiplos\n\n");
    printf("Entre com dois valores inteiros:\n");
    printf("A = ");
    scanf("%d", &a);
    printf("B = ");
    scanf("%d", &b);
    
    if (a<b){
        troca = b;
        b = a;
        a = troca;}
        
    multiplo = a %b;
    
    if (multiplo==0){
        printf("O número %d é multiplo de %d.\n",a,b);
    }else{  
        printf("Os números %d e %d não são múltiplos.\n", a,b);
    }
    
    printf("\nFim\n");
    return 0;
}
    

File: DAGF-AlgortimoProgramacaoDeComputadores-exercicios/cap06/multiplos.java
import java.io.*;
import java.util.Scanner;

class multiplos{
    public static void main(String args[]){
        int a,b,troca,resto;
        Scanner s = new Scanner(System.in);
        
        System.out.println("WADAJU SOFTWARE SOLUTIONS");
        System.out.println("Multiplos");
        System.out.println();
        System.out.println("Entre com dois valores inteiros:");
        System.out.print("A = ");
        a = s.nextInt();
        System.out.print("B = ");
        b = s.nextInt();
        
        if (a < b ) {
            troca = b;
            b = a ;
            a = troca;
            }
        
        resto = a % b ;
        
        if ( resto == 0 ) {
            System.out.println("O número "+a+" é multiplo do número "+b+".");
        } else{
            System.out.println("O número "+a+" não é multiplo de " +b+".");
            }
            
            System.out.println();
    //        return 0;
    }
}
